nltk/data.py

```diff
@@ -212,7 +217,8 @@ def normalize_resource_name(resource_name, allow_relative=True, relative_path=No
     >>> windows or normalize_resource_name('/dir/file', True, '/') == '/dir/file'
     True
     """
-    is_dir = bool(re.search(r'[\\/.]$', resource_name)) or resource_name.endswith(os.path.sep)
+    is_dir = bool(
+        re.search(r'[\\/.]$', resource_name)) or resource_name.endswith(os.path.sep)
```

Comment:
This change makes code less clear because the expression is split at a random point. With this indentation I'd expect bool() to be applied to the whole expression.